packages/infra/src/storage.ts

```diff
@@ -67,7 +67,7 @@ export class Storage extends Construct {
         securityHeadersBehavior: {
           contentSecurityPolicy: {
             contentSecurityPolicy:
-              "default-src 'none'; base-uri 'none'; form-action 'none'; frame-ancestors *.medplum.com;",
+              `default-src 'none'; base-uri 'none'; form-action 'none'; frame-ancestors *.${config.domainName};`,
```

Comment:
Fixes irrelevant hard-coded `medplum.com` to use the relevant domain name for content security. This allows the binary to be rendered in the medplum app hosted at app.medplum-test.letsdevelo.com

![image](https://user-images.githubusercontent.com/19141398/220670538-1223fd70-43e6-4540-a96d-10fa450aac96.png)

Reply:
There may be other places where we'd like to replace hardcoded "medplum". But this change seems to be enough for now

Reply:
good to know